lib/pages/main/main_vc.dart

```diff
@@ -138,97 +138,99 @@ class _MainVCState extends State<MainVC> {
-                                fontSize: 24),
-                            )
-                          );
-                        }
-                      ),
-                    ),
-                  ),
 
-                  SizedBox(width: 12),
+                        SizedBox(width: 12),
 
-                  Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: [
-                      Padding(
-                        padding: EdgeInsets.only(top: 50.0, bottom: 10.0),
-                        child: Text(
-                          MyLocalizations.of(context, 'welcome_text'),
-                          style: TextStyle(fontSize: 22),
-                        ),
-                      ),
-                      _uuidWithClipboard(),
-                    ],
-                  )
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Padding(
+                              padding: EdgeInsets.only(top: 50.0, bottom: 10.0),
+                              child: Text(
+                                MyLocalizations.of(context, 'welcome_text'),
+                                style: TextStyle(fontSize: 22),
+                              ),
+                            ),
+                            _uuidWithClipboard(),
+                          ],
+                        )
+                      ],
+                    )
+                  ),
+                  _buildCustomTile(0, Icons.home, 'home_tab', context),
+                  _buildCustomTile(1, Icons.file_copy, 'your_reports_txt', context),
+                  _buildCustomTile(2, Icons.map, 'public_map_tab', context),
+                  _buildCustomTile(3, Icons.biotech, 'guide_tab', context),
+                  _buildCustomTile(4, Icons.settings, 'settings_title', context),
+                  _buildCustomTile(5, Icons.info, 'info_tab', context),
                 ],
-              )
-            ),
-            _buildCustomTile(0, Icons.home, 'home_tab', context),
-            _buildCustomTile(1, Icons.file_copy, 'your_reports_txt', context),
-            _buildCustomTile(2, Icons.map, 'public_map_tab', context),
-            _buildCustomTile(3, Icons.biotech, 'guide_tab', context),
-            _buildCustomTile(4, Icons.settings, 'settings_title', context),
-            _buildCustomTile(5, Icons.info, 'info_tab', context),
-            SizedBox(
-              height: 60,
+              ),
             ),
-            Center(
-              child: Column(
-                mainAxisSize: MainAxisSize.max,
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: <Widget>[
-                  Text(packageInfo != null
+            Align(
+              alignment: Alignment.bottomCenter,
+              child: Padding(
+                padding: const EdgeInsets.only(bottom: 20.0),
```

Comment:
We can move it up or down from the absolute bottom by changing the value of `bottom` here